File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file132.f90
program DRB129_mergeable_taskwait_orig_yes
    use omp_lib
    implicit none

    integer :: x
    x = 2

    !$omp task mergeable
    x = x+1
    !$omp end task

    print 100, x
    100 format ('x =',3i8)
end program

File: data/gpt-4o/readable_format_HPEC/cpp/file132.cpp
#include <stdio.h>
#include <omp.h>

int main()
{
    int x = 2;

#pragma omp task mergeable
    {
        x = x+1;
    }
#pragma omp taskwait

    printf ("x = %d
", x);
    return 0;
}
